External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -1415,21 +1383,12 @@ private static void ConvertToConst(ClassModel inClass, ScintillaControl Sci, Mem
             int position = Sci.CurrentPos;
             MemberModel latest = null;
 
-            int wordPosEnd = Sci.WordEndPosition(position, true);
-            int wordPosStart = Sci.WordStartPosition(position, true);
-            char cr = (char)Sci.CharAt(wordPosEnd);
-            if (cr == '.')
-            {
-                wordPosEnd = Sci.WordEndPosition(wordPosEnd + 1, true);
-            }
-            else
-            {
-                cr = (char)Sci.CharAt(wordPosStart - 1);
-                if (cr == '.')
-                {
-                    wordPosStart = Sci.WordStartPosition(wordPosStart - 1, true);
-                }
-            }
+            int wordPosEnd = position + 1;
+            int wordPosStart = position;
+
+            while (Sci.BaseStyleAt(wordPosEnd) == style) wordPosEnd++;

```

Comment:
When the cursor is at the end of file, it seems like it returns 0 (`DEFAULT`) rather than throwing an exception. I figured it out while working on the auto closing braces.